src/main.rs

```diff
@@ -38,7 +42,26 @@ fn main() {
         match err {
             // Use default config when not found
             config::Error::NotFound => {
-                err_println!("Config file not found; using defaults");
+                env::var("HOME")
```

Comment:
This is getting a bit too verbose for `main.rs`. It would be nice if there was a method like `Config::write_defaults()` that encapsulated this logic.